x-pack/test/security_solution_cypress/cypress/screens/rule_details.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-export const ALL_ACTIONS = '[data-test-subj="rules-details-popover-button-icon"]';
+export const REST_ACTIONS_BUTTON = '[data-test-subj="rules-details-popover-button-icon"]';
```

Comment:
It wasn't immediately clear for me what `REST_ACTIONS_BUTTON` referred to before navigating to this file. Why the change of name from `ALL_ACTIONS`? Not sure I understand what you mean with `REST`